src/components/transport_manager/src/usb/libusb/usb_connection.cc

```diff
@@ -241,22 +250,35 @@ TransportAdapter::Error UsbConnection::SendData(
                       << "disconnecting_");
     return TransportAdapter::BAD_STATE;
   }
-  sync_primitives::AutoLock locker(out_messages_mutex_);
-  if (current_out_message_.use_count() != 0) {
-    out_messages_.push_back(message);
-  } else {
-    current_out_message_ = message;
-    if (!PostOutTransfer()) {
-      controller_->DataSendFailed(
-          device_uid_, app_handle_, message, DataSendError());
-      LOG4CXX_TRACE(
-          logger_,
-          "exit with TransportAdapter::FAIL. Condition: !PostOutTransfer()");
-      return TransportAdapter::FAIL;
+
+  auto process_message = [&]() {
```

Comment:
@SNiukalov Could we catch **this** only instead of catching by default?
```suggestion
  auto process_message = [this]() {
```

Reply:
@mvorobio 
Thanks, fixed in:https://github.com/smartdevicelink/sdl_core/pull/3332/commits/5904e97510a20cfd7b780f3906f2438231c7b5cb